src/exposedComponents/LabelBreakdown/LazyLabelBreakdown.tsx

```diff
@@ -2,7 +2,14 @@ import React, { lazy, Suspense } from 'react';
 
 import { type LabelBreakdownProps } from './LabelBreakdown';
 
-const LabelBreakdown = lazy(() => import('./LabelBreakdown'));
+const LabelBreakdown = lazy(async () => {
+  // Initialize i18n for scenes library
+  const { initPluginTranslations } = await import('@grafana/i18n');
+  const { loadResources } = await import('@grafana/scenes');
+  await initPluginTranslations('grafana-scenes', [loadResources]);
+
+  return import('./LabelBreakdown');
+});
```

Comment:
This is a bug fix for the legacy exposed component for the Knowledge Graph. The fix is included here for correctness only. I don't expect that users will see this component IRL.